shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Val.hs

```diff
@@ -19,6 +19,10 @@
 module Cardano.Ledger.Val
   ( Val (..),
     (~~),
+    (<+>),
```

Comment:
<+> <-> are a nice

Reply:
I also added   (a <×> b) for (scale a b)